src/tools/compiletest/src/runtest.rs

```diff
@@ -2710,6 +2712,25 @@ impl<'test> TestCx<'test> {
         fs::remove_dir(path)
     }
 
+    fn run_js_doc_test(&self) {
+        if let Some(nodejs) = &self.config.nodejs {
```

Comment:
Can we make this call `self.fatal("no nodeJS")` or some such? If we're in compiletest we should fail if we don't have a nodeJS here I think

Reply:
Ok!